juniper_official/Linecard/aft-sandbox-stats.yml

```diff
@@ -0,0 +1,47 @@
+SandboxStatsTable:
```

Comment:
1. According to guidelines, yml file 1st line should be with "---"
2. Fix below errors
    1:1 error too many blank lines (1 > 0) (empty-lines)
    2:1 warning missing document start "---" (document-start)
   14:81 warning line too long (115 > 80 characters) (line-length)
   24:81 warning line too long (151 > 80 characters) (line-length)

Reply:
These two comments cannot be fixed as it results in rule not matching the title field.

Reply:
14:81 warning line too long (115 > 80 characters) (line-length)
24:81 warning line too long (151 > 80 characters) (line-length)